src/main/scala/cz/alenkacz/db/pasquared/ConfigParser.scala

```diff
@@ -0,0 +1,68 @@
+package cz.alenkacz.db.pasquared
+
+import com.github.mauricio.async.db.Configuration
+import com.github.mauricio.async.db.pool.PoolConfiguration
+import com.github.mauricio.async.db.postgresql.util.URLParser
+import com.typesafe.config.Config
+
+import scala.concurrent.duration.Duration
+import scala.util.{Failure, Success, Try}
+
+object ConfigParser {
+  private final val MaxMessageSize = "maxMessageSize"
+  private final val ConnectTimeout = "connectTimeout"
+  private final val TestTimeout = "testTimeout"
+  private final val QueryTimeout = "queryTimeout"
+  private final val PoolConfig = "pool"
+  private final val PoolMaxConnections = "maxConnections"
+  private final val PoolIdleTime = "idleTime"
+  private final val PoolMaxQueueSize = "maxQueueSize"
+  private final val PoolValidationInterval = "validationInterval"
+
+  private val defaultConfig = URLParser.DEFAULT
+
+  def parse(config: Config): Try[PostgresConfiguration] = {
+    for {
+      c <- getConfiguration(config)
+      p <- getPoolConfiguration(config)
+    } yield PostgresConfiguration(c, p)
+  }
+
+  implicit def asFiniteDuration(d: java.time.Duration): Duration =
+    scala.concurrent.duration.Duration.fromNanos(d.toNanos)
+
+  private def getPoolConfiguration(config: Config): Try[Option[PoolConfiguration]] = {
+    if (config.hasPath(PoolConfig)) {
```

Comment:
hmm you are right. Then maybe the default config should not contain pool config? Do you really think that pooling postgresql connection is the preferred default choice?

In postgresql-async it is not the case...